app/src/main/java/ch/epfl/sdp/drone3d/ui/map/BaseMapActivity.kt

```diff
@@ -18,11 +20,13 @@ open class BaseMapActivity : AppCompatActivity() {
     lateinit var mapView: MapView
 
     protected fun initMapView(savedInstanceState: Bundle?, contentViewId: Int, mapViewId: Int) {
+        Mapbox.getInstance(this, getString(R.string.mapbox_access_token))
```

Comment:
If you put this line here, you should change all the activities that inherits from BaseMapActivity so that the same line of code is not called twice. Or even better, as the coaches wrote in their review, we should call it only once to get the instance of mapbox only once. However, I tried to put it in the main application (Drone3D) as it was suggested, but some tests didn't pass and I removed it, so I don't know if it's possible

Reply:
Yes indeed, I made all the required changes. I thought it was a good compromise for now, to call it only in the basemap activity so that it will be easier in the future to make the required changes to call it once in the app.